src/components/common/Stepper.vue

```diff
@@ -69,6 +29,147 @@ export default class Stepper extends Vue {
   @Getter isBusySaving!: GetterIF
   @Getter isTypeBcomp!: GetterIF
   @Getter isTypeCoop!: GetterIF
+
+  // State
+  @State(state => state.stateModel.defineCompanyStep.valid)
+  readonly step1Valid!: boolean;
+
+  // FUTURE - map these to states as above
+  readonly step2Valid!: boolean;
```

Comment:
Definitely like the mapping! It might get trickier once a step needs to map to more than one view. ie step4valid will be different based on entityType.

Reply:
Yes, but we can deal with that when we know more.